### PR TITLE
rearrange layout to be more columnar (src on left ; dest on right) in

### PR DESCRIPTION
**Pull request by eddiebroad**
the import/export MC screen.
Add refs to labels.
Add tempred function for making style color green for corresponding label
if error detected during a basic validation

### CODE REVIEW

**Comment by eddiebroad** [comment]
this error notification is basic.  It Might work for now, but it seems we need something more polished

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs [hunk 1]
```diff
@@ -43,9 +51,9 @@
                        n-basic-valid (if (>= dest-ws-n-len 1) true false)
                        ns-basic-valid (if (>= dest-ws-ns-len 1) true false)]
                    (if-not n-basic-valid
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs]: This line can be replaced with `(if (zero? (count (:name workspace-id)))` (and similarly for the other check), and then you can get rid of 4 lines in the `let`.

**Comment by MattPutnam** [comment]
I think this is a reasonable solution for now.  I've been thinking about creating a "validatable text field" component that would replace this (and a lot of others), but haven't gotten around to it.

Format to get rid of some extraneous whitespace and this gets a :+1: (unofficial review) from me.

**Comment by davidangb** [comment]
from a UX perspective, I don't like this approach. The validation messages are time-sensitive; if you attempt to import/export a config, look away for a moment, and the import/export failed, you will have no idea what happened. I'd prefer to keep the js alerts until we have time to think through the proper UX for this.

I'm fine with rearranging the columns.

**Comment by MattPutnam** [comment]
The better thing to do would be to set the color to red indefinitely, and change it back when the user types something into the text field.  This would take a bit more work though.

**Comment by eddiebroad** [comment]
@davidangb I commited code for column rearrangement based on a comment from Chet

**Comment by eddiebroad** [comment]
@MattPutnam Re: "The better thing to do...." I committed for that (05bf9....)

**Comment by davidangb** [comment]
So, I have no real problem with this code ... I just don't think it's worth it. The js alert errors are basic and simple and functional. This change feels like it's a partial step towards a better UI, but it incurs technical debt. For instance, I'd rather see the label coloring be completely dynamic:
* upon first loading, labels would be red because their values are empty
* as soon as a user enters a value, labels become black
* as soon as a user erases the value (backspace/cut/whatever), they become red again

Implementation-wise, this should be a purely dynamic color setting based on the value, not needing to call a separate function.

Bottom line, I think we should postpone these changes until we've had a chance to figure out the right approach.
